README.md

```diff
@@ -113,6 +113,19 @@ stripe := &client.API{}
 stripe.Init("access_token", nil)
 ```
 
+In order to make a request on behalf of a managed account, use the
```

Comment:
For this doc section, should we refactor to match the outline [here](https://stripe.com/docs/connect/authentication) of saying there are two ways to connect - with Stripe account IDs (suggested) and with access tokens, then provide details for each?

I'm not sure that it's still the case that a client is required, as mentioned in the older docs above?

I think the terminology we want here is "on behalf of a connected account" or "for a connected account", and not "of  a managed account", which means something a bit [more specific](https://stripe.com/docs/connect/managed-accounts).